almalinux-deploy.sh

```diff
@@ -360,6 +362,25 @@ grub_update() {
     fi
 }
 
+# Check do we have custom kernel (e.g. kernel-uek) and print warning
+check_custom_kernel() {
+    local output
+    output=$(rpm -qa | grep kernel-uek) || :
+    if [ -n "${output}" ]; then
+        if [ -x /usr/bin/mokutil ] && /usr/bin/mokutil --sb-state 2>&1 | grep -q enabled; then
+            echo -ne "\n!! [31;1mYou've had kernels that can't be booted in
```

Comment:
There are kernels left from previous operating system that won't boot in Secure Boot mode anymore: